src/permission/types.py

```diff
@@ -31,6 +33,21 @@ async def has_permission(db_session: database.DBSession, computing_id: str) -> b
 
         return most_recent_exec_term.end_date > cutoff_date
 
+class ElectionOfficer:
+    @staticmethod
+    async def has_permission(db_session: database.DBSession, computing_id: str) -> bool:
+        """
+        An current elections officer has access to all elections, prior elections officers have no access.
+        """
+        officer_terms = await officers.crud.current_executive_team(db_session, True)
```

Comment:
`current_executive_team` may enforce certain permissions in order to get private data from other people (TODO: double check this gabe). I created a function in https://github.com/CSSS/csss-site-backend/pull/86/commits/746e92eaffec3c9573eb17c9069b0e71fe505e02 to get the list of positions a single officer has, which would be better for this, though depends on my issue getting completed...